tests/commands/load/test_load_case_cmd.py

```diff
@@ -116,25 +98,19 @@ def test_load_case_SyntaxError(mock_app, institute_obj, case_obj, monkeypatch):
     def mock_parse_case(*args):
```

Comment:
I think it would be much more useful for testing purposes is the error is naturally triggered by a faulty config file, instead of a mocker that triggers the error like this

Reply:
This error is mostly hypothetical. Any syntax error in the configuration will be caught by the yaml parser and secondly by the pydantic parser. I haven't come up with a way to manipulate the config file to get through.